### PR TITLE
Add dependencies for native host

### PR DESCRIPTION
**Pull request by bbrto21**
Fixes the build failure mentioned in https://github.com/flutter-tizen/flutter-tizen/pull/379#issuecomment-1169421471.

### CODE REVIEW

File: lib/build_targets/package.dart [hunk 1]
```diff
@@ -295,6 +295,8 @@ class NativeTpk extends TizenPackage {
       'capi-appfw-app-common',
       'capi-appfw-application',
       'dlog',
+      'elementary',
```
Comment (swift-kim) [inline lib/build_targets/package.dart:298]: Is `elementary` for future use?

**Review by swift-kim — APPROVED** [review]

File: lib/build_targets/package.dart (hunk 1 above)
Comment (bbrto21) [reply lib/build_targets/package.dart:298]: Since Elemental.h is explicitly included in elm_flutter_view.h, I added it. should I except it?

Comment (swift-kim) [reply lib/build_targets/package.dart:298]: Not really. It might be useful in the future.

**Comment by bbrto21** [comment]
We can't build a flutter-tizen app using native app host now, so I want to merge it right away. can I do that?

**Pull request merged**